### PR TITLE
Update overview page top section

### PR DESCRIPTION
**Pull request by ViridianCitrus**

### CODE REVIEW

File: src/components/Minecraft/MinecraftOverview.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+  return (
+    <>
+      <div
+        className="relative flex h-full w-full max-w-2xl flex-col justify-center @container"
+        key={uuid}
+      >
+        {/* main content container */}
+        <div className="flex w-full grow flex-col items-stretch gap-2 px-4 ">
+          <div className="flex w-full min-w-0 flex-row items-center gap-4">
```
[the remark this answers is not in the document]
Comment (Ynng) [reply src/components/Minecraft/MinecraftOverview.tsx:69]: image
Also the `Overview` title is still there.
can you get rid of it?

Maybe in the tab list map, add a new field "display title" so we can set it to like nothing, or something else that works.

**Review by Ynng — CHANGES_REQUESTED** [review]

File: src/components/ClipboardTextfield.tsx [hunk 2]
```diff
@@ -32,12 +36,18 @@ export default function ClipboardTextfield({
         onClick={onClickCopy}
       >
         <div className={`select-none`}>
+          {/* TODO develop custom tooltip component */}
+          {text}&nbsp;&nbsp;
```
Comment (Ynng) [inline src/components/ClipboardTextfield.tsx:40]: The copy ip button should have the text to the right, and icon to the left.
Maybe make `iconSide` a prop too?

Also the icon currently have `mr-2` but if the text is on the left, the space is in the wrong place. please look into tailwindcss `gap` or `space`

image

**Review by Ynng — CHANGES_REQUESTED** [review]
image
tl;dr: copy ip button looks weird

File: src/components/ClipboardTextfield.tsx [hunk 3]
```diff
@@ -37,17 +39,22 @@ export default function ClipboardTextfield({
...
-              red: 'mr-2 text-red-faded/25 group-hover:text-red',
-            }[color])}
+              gray: `text-gray-faded/25 group-hover:text-gray-500`,
+              blue: `text-blue-faded/25 group-hover:text-blue-150`,
+              green: 'text-green-faded/25 group-hover:text-green',
+              yellow: 'text-yellow-faded/25 group-hover:text-yellow',
+              red: 'text-red-faded/25 group-hover:text-red',
+            }[color],
+            {
+              'mr-2': iconLeft,
```
Comment (Ynng) [inline src/components/ClipboardTextfield.tsx:52]: I think we need `ml-2` if icon left is false

**Review by Ynng — CHANGES_REQUESTED** [review]

**Review by Ynng — CHANGES_REQUESTED** [review]
We need to get rid of the two non breaking space `&nbsp;&nbsp;` too, they are messing with the spacing

**Review by Ynng — APPROVED** [review]
lgtm!
